contrib/libmfx/module.defs

```diff
@@ -6,7 +6,9 @@ LIBMFX.FETCH.sha256 = e07d1024e86998ac3992620f5db0f999af51cc700f7de90da2ebbe1e8a
 
 LIBMFX.CONFIGURE.bootstrap = rm -fr aclocal.m4 autom4te.cache; autoreconf -fiv;
 
+ifneq (1,$(FEATURE.qsv))
```

Comment:
@bradleysepos can help as well due to https://github.com/HandBrake/HandBrake/commit/77bdefe5a4ab639460bfcb3acec3f77ab7e414c0

General rule:
- it should not matter for cross/Windows compilation
- for Linux we need to have it

Note these (libva_drm and libva_x11) should be from MSS and not other(s)/default ones.